src/instructlab/model/train.py

```diff
@@ -560,6 +560,25 @@ def get_files(directory: str, pattern: str) -> list[str]:
         # try to load journal. May be empty.
         journal = TrainingJournal(journalfile=training_journal)
         click.secho("\n\n~~~~~~~~~~~~~STARTING MULTI-PHASE TRAINING~~~~~~~~~~~~~")
+
+        # experimental preference.
+        if phased_phase1_num_epochs != 7:
+            click.secho(
+                "Experimentally, Phase 1 of training is most effective after '7' epochs.",
+                fg="black",
+                bg="yellow",
+            )
+            click.secho(
+                f"'{phased_phase1_num_epochs}' epochs were configured.",
+                fg="black",
+                bg="yellow",
+            )
+            click.secho(
+                f"'{phased_phase1_num_epochs}' This can result in sub-optimal final checkpoint performance.",
```

Comment:
Is '{phased_phase1_num_epochs}' supposed to be at the beginning of this sentence?

Reply:
It was, but on consideration I don't think that's the right move.